### DISCUSSION TITLE
How to run a Linux Command in the Background with Deno.run

### DISCUSSION THREAD

**Discussion by ynwd** [denoland/deno#14055] [Q&A]
I want to try run `deno run -A deno.ts &` using `Deno.run`

```ts
const cmd = ["deno", "run", "-A", "deno.ts", "&"];

const p = Deno.run({ cmd });

await p.status();
```

but it doesn't work.

Is there anything missing?

**Comment by crowlKats** [comment]
`&` isnt handled as you expect it to, as thats something handled by a shell. What exactly are you trying to achieve?

**Comment by ynwd** [comment]
i want to implement this node.js based code in deno.

```js
const { execSync } = require("child_process");

execSync(`deno run -A deno.ts &`, { stdio: [0, 1, 2], cwd: process.cwd() });
```

**Comment by ghost** [comment]
You should be able to simply remove the shell syntax, and ignore the result, which would let it run completely in the background:
```ts
const cmd = ["deno", "run", "-A", "deno.ts"];

Deno.run({ cmd });
```
This is what the Node.js example did too - it simply ignored it; `execSync`'s synchronicity was countered by the shell `&`.
